bin/intkey-tp-rust

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+#
+# Copyright 2018 Bitwise IO, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# -----------------------------------------------------------------------------
+
+top_dir=$(cd $(dirname $(dirname $0)) && pwd)
+cd $top_dir/sdk/examples/intkey_rust
+
+exec cargo run --release -- $*
```

Comment:
This might not be the best thing to do here, since this currently triggers a full build, due to source generation from the proto files.  Something to look into soon after this is merged.

Reply:
I will add it to my todo list.